cristin-import/src/test/resources/application.conf

```diff
@@ -0,0 +1,5 @@
+doi {
```

Comment:
Lightbend config uses the following configuration hierarchy from lowest to highest:
1. reference.conf
2. application.conf overrides reference.conf
3. system properties overrides application.conf

We can view the reference  configuration and the application configuration as a custom configuration per application. Since the the tests are in a way an application of the code we have a reference file in the main code and an application file in the test code.